dss/events/handlers/index.py

```diff
@@ -1,36 +1,56 @@
+
+
+"""Lambda function for DSS indexing"""
+
 import json
 import os
 import re
+import uuid
 from urllib.parse import unquote
 
 import boto3
+import botocore
+import requests
+from elasticsearch.helpers import scan
 
-from ... import DSS_ELASTICSEARCH_INDEX_NAME, DSS_ELASTICSEARCH_DOC_TYPE
-from ...util import connect_elasticsearch
+from dss import (DSS_ELASTICSEARCH_INDEX_NAME, DSS_ELASTICSEARCH_DOC_TYPE,
```

Comment:
Yes, a good idea. Let's do that as a separate PR, though (we have a lot, too much, in-flight already :)